nav2_rviz_plugins/include/nav2_rviz_plugins/goal_tool.hpp

```diff
@@ -36,7 +35,7 @@ class StringProperty;
 namespace nav2_rviz_plugins
 {
 
-class RVIZ_DEFAULT_PLUGINS_PUBLIC GoalTool : public rviz_default_plugins::tools::PoseTool
```

Comment:
Why was this removed?

Reply:
See https://github.com/ros-navigation/navigation2/pull/4868 .